src/openqaoa-braket/backends/devices.py

```diff
@@ -0,0 +1,149 @@
+import numpy as np
+
+from boto3.session import Session
+from botocore.exceptions import NoRegionError
+from braket.aws import AwsDevice
+from braket.aws.aws_session import AwsSession
+
+from openqaoa.backends.devices_core import DeviceBase
+
+
+class DeviceAWS(DeviceBase):
+
+    """
+    Contains the required information and methods needed to access QPUs hosted
+    on AWS Braket.
+
+    Attributes
+    ----------
+        available_qpus: `list`
```

Comment:
Wrong indent

Reply:
Will be changed in a newer commit.

Reply:
Resolved in commit [857b62e](https://github.com/entropicalabs/openqaoa/pull/170/commits/857b62ecc3682c048048e207bbd372272cf11462)